.github/workflows/ci-artifacts.yml

```diff
@@ -0,0 +1,236 @@
+        shell: bash
+        run: |
+          for path in clangarm64/bin/7z.exe clangarm64/bin/7z.dll clangarm64/lib/7zip/7zCon.sfx clangarm64/bin/libc++.dll
+          do
+            git --git-dir=git-sdk-arm64.git show HEAD:$path >${path##*/}
+          done &&
+          mkdir minimal-sdk-extra &&
+          (cd minimal-sdk && ../7z.exe a -mmt=on -mx9 ../minimal-sdk-extra/git-sdk-aarch64-minimal.zip * .?*) &&
+          (cd minimal-sdk && ../7z.exe a -t7z -mmt=on -m0=lzma -mqs -mlc=8 -mx=9 -md=256M -mfb=273 -ms=256M -sfx../7zCon.sfx \
+            ../minimal-sdk-extra/git-sdk-aarch64-minimal.7z.exe * .?*)
+      - name: upload minimal-sdk-extra artifacts
+        # if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        if: github.event_name == 'push' && github.ref == 'refs/heads/add-ci-artifacts'
+        uses: actions/upload-artifact@v4
+        with:
+          name: minimal-sdk-extra
+          path: minimal-sdk-extra
+  assorted-validations:
+    runs-on: dennis-windows-arm64
+    needs: [minimal-sdk-artifact]
+    steps:
+      - name: TO-DROP Install build dependencies
+        shell: powershell
+        run: |
+          # Download the script to a local path
+          Invoke-WebRequest -Uri "https://raw.githubusercontent.com/dennisameling-org/git-for-windows-automation/github-hosted-runners/azure-self-hosted-runners/post-deployment-script.ps1" -OutFile "$env:TEMP\post-deployment-script.ps1"
+
+          # Set PowerShell execution policy to unrestricted for the current session
+          Set-ExecutionPolicy -Scope Process -ExecutionPolicy Unrestricted
+
+          # Run the script
+          & "$env:TEMP\post-deployment-script.ps1"
+          
+          Add-Content $env:GITHUB_PATH "C:\Program Files\Git\bin"
+          Add-Content $env:GITHUB_PATH "C:\Program Files\Git\cmd"
+          Add-Content $env:GITHUB_PATH "C:\Program Files\PowerShell\7"
+      - name: download minimal-sdk artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: minimal-sdk
+          path: ${{github.workspace}}
+      - name: uncompress minimal-sdk
+        shell: bash
+        run: |
+          mkdir -p minimal-sdk &&
+          tar -C minimal-sdk -xzf git-sdk-aarch64-minimal.tar.gz &&
+          cygpath -aw minimal-sdk/usr/bin >>$GITHUB_PATH
+      - name: run some tests
+        shell: bash
+        run: |
+          set -x
+          . /etc/profile
+
+          # cygpath works
+          test "$(cygpath -aw /)" = "${{github.workspace}}\minimal-sdk" || exit 1
+
+          # comes with Clang and can compile a DLL
+          test "$(type -p clang)" = "/clangarm64/bin/clang" || exit 1
+          cat >dll.c <<-\EOF &&
+          __attribute__((dllexport)) int increment(int i)
+          {
+              return i + 1;
+          }
+          EOF
+
+          clang -Wall -g -O2 -shared -o sample.dll dll.c || exit 1
+          ls -la

```

Comment:
This currently [fails](https://github.com/dennisameling-org/git-sdk-arm64/actions/runs/10862913847/job/30146598856#step:5:50), apparently because some dependencies of `clang.exe` are missing in `minimal-sdk`:

<img width="440" alt="Screenshot 2024-09-14 at 22 25 02" src="https://github.com/user-attachments/assets/b34f7b79-2b21-47f1-a680-d3d47cae71df">

This is the complete list of files missing in `minimal-sdk`:

```
clangarm64/bin/libffi-8.dll
clangarm64/bin/libxml2-2.dll
clangarm64/bin/liblzma-5.dll
```

I'll file a separate PR for [`.sparse/minimal-sdk`](https://github.com/git-for-windows/git-sdk-arm64/blob/main/.sparse/minimal-sdk) to add the missing files.

Reply:
PR to fix: https://github.com/git-for-windows/git-sdk-arm64/pull/23